### ISSUE TITLE
Newer openjdk / postgresql packages

### ISSUE THREAD

**Issue by padey** [nzymedefense/nzyme#1116]
Hi Lennar,

Since “openjdk-11-jre-headless postgresql-14” and are quite old, they can no longer be installed under Debian 12, for example. Is there a possibility that newer versions can be used here?

Ubuntu has become somewhat difficult with its license policy... I'm trying to get node&tap running on a NanoPi R6C and there is only Debian 12 a good OS. :/

Best regards
Patrick

**Comment by lennartkoopmann** [comment]
Hi Patrick,

I understand. There is #557 but it has been delayed because Raspberry Pi OS was not shipping with any recent enough Java Runtime Environments. I believe this has changed and I re-prioritized the issue.

OpenJDK 11 will be EOL in just about 2 months, so we have to do something anyways. We will move to the latest LTE version that Raspberry Pi OS offers. (simply because of how widespread Raspberry Pi usage is in the nzyme community)

If possible, go with a manual install of a JRE 11 for now?

More recent PostgreSQL versions are already well supported by nzyme. You can go ahead and run those as you wish. The only change is that you have to add additional permissions to the `public` schema since version 15. See also: https://stackoverflow.com/a/75876944

Hope that helps!

Thanks,
Lennart

**Comment by padey** [comment]
Hi Lennart. 

many thanks for the fast response. I tried the manual installation but all the required packages need to be installed manually. I'm gonna try it again on Monday :) 


Gruß/Best
Patrick

**Comment by lennartkoopmann** [comment]
Yes, there will be official RPI OS / Debian 12 packages and docs soon.

**Comment by padey** [comment]
Nice @lennartkoopmann !

**Comment by lennartkoopmann** [comment]
Packages are available with the latest alpha. Thanks!
